arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1895,11 +1903,11 @@ Result RocksDBCollection::lookupDocument(transaction::Methods& trx,
     return rocksutils::convertStatus(s, rocksutils::document);
   }
 
-  if (fillCache && useCache() && !lockTimeout) {
-    TRI_ASSERT(_cache != nullptr);
+  if (fillCache && cache != nullptr && !lockTimeout) {
+    TRI_ASSERT(cache != nullptr);
```

Comment:
We checked this condition in if
```suggestion
  if (fillCache && cache != nullptr && !lockTimeout) {
```